public/javascripts/Gallery/css/cards.css

```diff
@@ -149,11 +150,18 @@ with respect to the image holder. This allows the validation menu to be placed o
     width: 100%;
     flex-direction: row;
     flex-wrap: wrap;
+    flex-wrap: nowrap; /* Prevent wrapping */
+    overflow: hidden; /*new*/
 }
 
 .label-tags-content {
-    text-overflow: ellipsis;
+    /* text-overflow: ellipsis;
```

Comment:
Feel free to just comment stuff out while you're testing, but before you submit the PR for review you should actually delete the stuff instead of having it commented out :)